src/Nethermind/Nethermind.Core2/Types/Epoch.cs

```diff
@@ -90,6 +90,8 @@ public override string ToString()
         public static implicit operator Epoch(ulong value) => new Epoch(value);
 
         public static implicit operator ulong(Epoch slot) => slot.Number;
+        
+        public static implicit operator int(Epoch slot) => (int)slot.Number;
```

Comment:
Same as other comments re implict / explict

Reply:
yup, this was meant to be explicit - made it consistent now